data/core/units/undead/Skele_Bone_Shooter.cfg

```diff
@@ -13,7 +13,7 @@
     alignment=chaotic
     advances_to=Banebow
     cost=26
-    description= _ "Of a dark sorcerer’s creations, some take more strongly to the false life given them. The potency of their un-life is given equipment to match; archers, in particular, are often outfitted with a truly vile arsenal. Their quivers are filled with shafts made not of wood, but of the bones of their victims. It follows that they are dubbed simply ‘Bone-Shooters’ by their unfortunate enemies."
+    description= _ "Of a dark sorcerer’s creations, some take more strongly to the false life given them. The potency of their unlife is given equipment to match; archers, in particular, are often outfitted with a truly vile arsenal. Their quivers are filled with shafts made not of wood, but of the bones of their victims. It follows that they are dubbed simply ‘Bone-Shooters’ by their unfortunate enemies."
```

Comment:
I checked for other instances of "unlife" and "un-life", and Wesnoth canon seems to be equally split between the two spellings (as in, there are more corrections needed whichever one we decide to use).

@nemaara which of the two spellings should be our house-style?

Reply:
I prefer unlife, it reads cleaner. Extra punctuation in unnecessary places is hard to read for me personally.